### PR TITLE
ROX-20650: split central test into two

### PR DESCRIPTION
**Pull request by porridge**
## Description

Since tests are run in parallel, splitting `central-basic` into two tests in theory speeds it up 2x. In practice (see timings below) the initial setup time dominates, so the speedup is much more modest. However it paves the way to supporting more cases in the future without slowing things down. This might be needed soon to support scanner v4.

This change also:
- bumps the number of worker nodes to make sure the extra central components fit, 
- updates documentation and 
- renames directories such that tests in a suite have a common prefix for consistency.

## GKE:

Before:
```
--- PASS: kuttl (889.27s)
[...]
        --- PASS: kuttl/harness/basic-central (886.53s)
```

After:
```
--- PASS: kuttl (744.32s)
[...]
--- PASS: kuttl/harness/central-misc (680.77s)
--- PASS: kuttl/harness/central-basic (741.58s)
```

## OCP:

Before:
```
--- PASS: kuttl (1231.60s)
[...]
        --- PASS: kuttl/harness/basic-central (1223.60s)
```

After:
```
--- PASS: kuttl (893.97s)
[...]
--- PASS: kuttl/harness/central-basic (760.86s)
--- PASS: kuttl/harness/central-misc (886.01s)
```

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~
- [x] ~Evaluated and added CHANGELOG entry if required~
- [x] ~Determined and documented upgrade steps~
- [x] ~Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

### Here I tell how I validated my change

Timings gathered from CI show a change in the desired direction.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

**Comment by porridge** [comment]
Rebasing to pick up https://github.com/stackrox/stackrox/commit/374923eb6ab8ff8c1ac3b614745da14002db1976

**Review by johannes94 — APPROVED** [review]
LGTM.

File: operator/tests/TROUBLESHOOTING_E2E_TESTS.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 ## Structure
 
-All kuttl tests are in the [tests](.) directory. It contains three suites, each containing a single test:
-- [central/basic-central](central/basic-central) - various `Central` resource configurations and switching between them.
-  In theory, on a large enough cluster this could be split into multiple parallel tests.
-- [securedcluster/basic-sc](securedcluster/basic-sc) - various `SecuredCluster` resource configurations
+All kuttl tests are in the [tests](.) directory. It contains four tests, contained in three suites:
```
[the remark this answers is not in the document]
Comment (porridge) [reply operator/tests/TROUBLESHOOTING_E2E_TESTS.md:5]: Maybe once we build some structure we can change this to a less detailed list. So far there were only two changes: this split, and introduction of the upgrade test.

**Pull request merged**